docs/src/main/sphinx/release/release-436.md

```diff
@@ -0,0 +1,57 @@
+# Release 436 (10 Jan 2024)
+
+## General
+
+* {{breaking}} Require JDK 21.0.1 to run Trino. ({issue}`20010`)
+* Improve performance by not generating redundant predicates. ({issue}`16520`)
+* Fix query failure when invoking the `json_table` function. ({issue}`20122`)
+* Fix query hang when a [SQL routine](/routines) dereferences a row field. ({issue}`19997`).
+* Fix potential incorrect results when using the {func}`ST_Centroid` and
+  {func}`ST_Buffer` functions for tiny geometries. ({issue}`20237`)
+
+## Delta Lake connector
+
+* Add the `parquet_ignore_statistics` catalog session property, which can be
```

Comment:
Allow control of querying files with corrupted or incorrect statistics with the new parquet_ignore_statistics catalog session property